Scripts/ISAN/test.yaml

```diff
@@ -0,0 +1,4 @@
+scripts: 
+  - Mono.yolol
+cases: []
```

Comment:
Did you try to write tests and they didn't work, or did you just not feel like adding test?
(Adding proper test-cases for ISAN probably isnt the simplest thing)

Reply:
I didn't try to write tests, since I'm just porting across ISAN directly from the doc.